roles/bridge-setup/tasks/main.yml

```diff
@@ -1,9 +1,31 @@
 ---
 
-- name: Install bridge-utils
+- name: Install bridge-utils and iptables
   yum:
-    name: bridge-utils
+    name: "{{ item }}"
     state: latest
+  with_items:
+    - bridge-utils
+    - iptables
+    - iptables-services
+
+- name: enable iptables
+  service:
+    name: iptables
+    state: started
+    enabled: yes
+
+- name: do iptables (to accept bridge forwarding and accept packets)
+  shell: >
+    iptables -I FORWARD -m physdev --physdev-is-bridged -j ACCEPT;
+    iptables -D INPUT -j REJECT --reject-with icmp-host-prohibited
+
+- name: enable ipv4 forwarding
```

Comment:
Minor nit, but can you follow the same upper-case leading format as the other `name` uses in the playbooks? e.g. `Enable IPv4 forwarding`

Reply:
changed.